vignettes/Data_retrieval.Rmd

```diff
@@ -498,31 +498,103 @@ mmthds <- mc_methods(aeids)
 mmthds
 ```
 
-## Plot MC Data
+# Plotting
 
-The <font face="CMTT10">tcplPlot</font> function is the generic plotting function that returns a plot for the given sample-assay endpoint combination(s), level, and parameters.
+<font face="CMTT10">tcplPlot</font> is tcpl’s single flexible plotting function, allowing for interactive yet consistent visualization of concentration-response curves via customizable parameters. As a standalone plotting utility built with the R library <font face="CMTT10">plotly</font> to display the additional curve-fitting models, <font face="CMTT10">tcplPlot</font> implements the R library <font face="CMTT10">plumber</font> to provide representational state transfer-application programming interface (<font face="CMTT10">REST API</font>) functionality. The <font face="CMTT10">tcplPlot</font> function requires the selection of a level (`lvl`), field (`fld`), and value (`val`) to load the necessary data and display the associated plots. Level 4, `lvl = 4`, plots the concentration-response series fit by all models. Level 5, `lvl = 5`, extends Level 4 plotting by highlighting the winning model with activity hit call presented. Level 6 multi-concentration and single-concentration plotting are not currently supported by <font face="CMTT10">tcplPlot</font>. 
 
-```{r mc_plots, eval = FALSE}
-# Create Level 4 plot for a single m4id.
-tcplPlot(lvl = 4,  # data level
-         fld = "m4id", # field to query on 
-         val = c(18609966), # value for each field
-                            # values should match their corresponding 'fld'
-         multi = FALSE, # single plot per page - output 6 per page if TRUE
-         verbose = FALSE, # output all details if TRUE
-         output = "console") # output in R console
+Customization of output is possible by specifying parameters, including `output`, `multi`, `fileprefix`, `by`, `verbose`, `nrow`, `ncol`, and `dpi`.  
+
+- The `output` parameter indicates how the plots will be presented. In addition to outputs viewable with the R console, <font face="CMTT10">tcplPlot</font> supports a variety of publication-quality file type options, including raster graphics (PNG, JPG, and TIFF) to retain color quality when printing to photograph and vector graphics (SVG and PDF) to retain image resolution when scaled to large formats.
+
+- The `multi` parameter allows for single or multiple plots per page. `Multi = TRUE` is the default option for PDF outputs, whereas `multi = FALSE` is the only option for other outputs. If using the parameter option `multi = TRUE` output is generated with either 6 plots per page (`verbose = FALSE`) or 4 plots per page (`verbose = TRUE`).
```

Comment:
May be worth a brief explanation as to why the 'verbose' argument correlates with the number of resulting plots per page.  I presume this is due to more "verbose" information in the graphs; however, this may not be evident to new users. Alternatively, you can just reference the suggestion in the next comment.